FirebaseAuth/Sources/Backend/RPC/FIRGetRecaptchaConfigRequest.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import "FirebaseAuth/Sources/Backend/FIRAuthRPCRequest.h"
+#import "FirebaseAuth/Sources/Backend/FIRIdentityToolkitRequest.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** @class FIRGetRecaptchaConfigRequest
+    @brief Represents the parameters for the getRecaptchaConfig endpoint.
+ */
+@interface FIRGetRecaptchaConfigRequest : FIRIdentityToolkitRequest <FIRAuthRPCRequest>
+
+/** @property clientType
+    @brief The client type should always be "ios".
+ */
+@property(nonatomic, copy, nullable) NSString *clientType;
+
+/** @property version
+    @brief The version of the reCAPTCHA service should be always be "enterprise".
+ */
+@property(nonatomic, copy, nullable) NSString *version;
+
+/** @fn initWithEndpoint:requestConfiguration:
+    @brief Please use initWithTenantId:clientType:version:requestConfiguration:
```

Comment:
Where is the `initWithTenantId:clientType:version:requestConfiguration:` method defined? The below method is missing the `tenantId` param.

Reply:
No tenantId needed in this method. It's automatically added here https://github.com/firebase/firebase-ios-sdk/blob/bdac2b87f6d6e604deae914a970a75136b3a62b5/FirebaseAuth/Sources/Backend/FIRIdentityToolkitRequest.m#L60
Updated `@brief`.